src/base/message.rs

```diff
@@ -665,6 +666,13 @@ impl<Octs: Octets + ?Sized> Message<Octs> {
     }
 }
 
+/// # Printing
+impl<Octs: AsRef<[u8]>> Message<Octs> {
+    pub fn display_dig_style(&self) -> impl core::fmt::Display + '_ {
```

Comment:
Small thing, but could you add a doc comment here?  It's part of the public API, after all.

Reply:
Fair enough :)